routers/actionRouter.js

```diff
@@ -0,0 +1,94 @@
+const express = require("express");
+const router = express.Router();
+
+const actions = require("../data/helpers/actionModel");
+
+router.post("/", (req, res) => {
+    const {project_id, description, notes, completed } = req.body;
+    if(!project_id, !description || !completed || !notes){
+        res.status(401).json({message: "could not add action"})
+    }
+    else if (description.length > 128){
+        res.status(401).json({message: "description must ne less then 128 characters"})
+    }
+    actions
+        .insert({project_id, description, notes, completed})
+        .then(response => {
+            res.status(201).json(response)
+        })
+        .catch(err => {
+            res.status(500).json({message: "error adding action"})
+        })
+})
+
+router.get("/", (req, res) => {
+    actions
+        .get()
+        .then(action => {
+            res.status(200).json(action);
+        })
+        .catch (err => {
+            res.status(500).json({message:"project not found"});
+        })
+})
+
+router.get("/:id", (req, res) => {
+    const {id} = req.params;
+    actions
+        .get(id)
+        .then(action => {
+            if(action){
+                res.json(action);
+            }
+            else {
+                res.status(404).json({message: "action does not exist"})
+            }
+        })
+        .catch (err => {
+            res.status(500).json({message: "action could not be retrieved"})
+        })
+})
+
+router.delete("/:id", (req, res) => {
+    const {id} = req.params;
+    actions
+        .remove(id)
+        .then(action => {
+            if(action){
```

Comment:
This is more of a style preference thing, but I think adding a space after the if keyword makes the statement more readable 👍 
```suggestion
            if (action){
```